ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml

```diff
@@ -373,6 +373,51 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: build02
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 5
```

Comment:
1. Not sure what `max_concurrency` is responsible for, but this attribute is absent in pull-request counterpart of this job.
2. There's 5 hours timeout in pull-request counterpart of the job. Do you want to add that here too?

Here's what I looked at:
https://github.com/openshift/release/blob/662657a1ecd4b05a4c35dd68843ce34e1e71459a/ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml#L730

Reply:
Just double checked and none of the `merge-` tests have a timeout even when the non `merge-` versions do so it's more consistent not to add one now.